src/main.py

```diff
@@ -29,6 +29,9 @@ def __init__(self, *args, **kwargs):
                          **kwargs)
         self.window = None
         self.file_path = None
+        GLib.set_application_name('Easy eBook Viewer')
+        GLib.set_prgname('easy-ebook-viewer')
+        GLib.setenv('PULSE_PROP_application.icon_name', 'easy-ebook-viewer', True)
```

Comment:
[`GLib.set_application_name`](https://lazka.github.io/pgi-docs/index.html#GLib-2.0/functions.html#GLib.set_application_name) should already set the PulseAudio name: https://www.freedesktop.org/wiki/Software/PulseAudio/Documentation/Developer/Clients/ApplicationProperties/#pa_prop_application_name

Reply:
Oh shit, this is not an audio software xD.
We need to remove that line.